websrc/source/documents/community/CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: "documents"
+page_title: "Contiv Community"
+sidebar_current: "community"
+description: |-
+  Contributing to Contiv
+---
+# Contributing to Contiv
+
+Submitting code changes is only one of the many ways to contribute.
+Reporting issues, proposing documentation and design changes,
+discussing use cases and proposing integration with other software
+from the ecosystem are more ways to contribute to Contiv.
+Contributors can also become maintainers or committers.
+
+All contributions are welcome, no matter how small or how big they are.
+
+Various repos may have more specific requirements for contribution - check out CONTRIBUTING.md
+files in respective repos.
+
+## Reporting Issues
+You can report an issue in a specific repo by 
+clicking `New Issue` in the respective repo.
+Issues should also be opened for feature requests.
+
+## Submitting pull requests for code or documentation changes
+Changes can be proposed by sending a pull request (PR). One of the committers
+will review the changes and provide feedback. The pull request will be merged
+into the master branch after discussion.
+
+Please make sure that the unit tests and the system tests pass before
```

Comment:
Not every project has tests, and not every one has unit/system tests.  Maybe:

"Please make sure that all tests are passing, if applicable"

We also have CI gates which prevent merging code unless tests are passing, anyways.